### PR TITLE
HCALDQM: Remove all conditions access except emap (TEMPORARY PR)

### PR DESCRIPTION
**Pull request by DryRun**
For MWGR1, HCAL conditions are not yet ready. This is a *temporary PR for online DQM* which removes all calls to conditions except for the emap, for which a tag has specially been prepared [1]. Note that only the "hcal" application will work, not "hcal2," which uses reco. 

[1]
-File: /afs/cern.ch/user/c/cawest/public/HCALemap_all_2018.txt
-Tag: HcalElectronicsMap_all_2018
-Payload: bf8532b2e18d42712a5b995da11a1edfd2bd779b

### CODE REVIEW

File: DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -12,31 +12,32 @@
...
 warnstr			= "### HcalDQM::cfg::WARN: "
 errorstr		= "### HcalDQM::cfg::ERROR:"
-useOfflineGT	= False
+useOfflineGT	= True
 useFileInput	= False
 useMap		= False
 
+# 2018 MWGR preparation: use text emap and a local run test file
+MWGR1_hacks = True
+useLocalFileInput = True
```
Comment (deguio) [inline DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py]: I think this one needs to be `False`

File: DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py [hunk 2]
```diff
@@ -50,17 +51,24 @@
 process.DQMStore.referenceFileName = referenceFileName
 process = customise(process)
 process.DQMStore.verbose = 0
-process.source.minEventsPerLumi=100
```
Comment (deguio) [inline DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py]: you might want to restore this one

Comment (DryRun) [reply DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py]: Hi Fede, thanks for catching this, should be fixed now.

File: DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py [hunk 3]
```diff
@@ -177,9 +252,10 @@
 #	Paths/Sequences Definitions
 #-------------------------------------
 process.preRecoPath = cms.Path(
-		process.hcalDigis
-		*process.castorDigis
-		*process.emulTPDigis
+		process.tbunpacker
```
Comment (deguio) [inline DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py]: I would remove this one for global runs

Comment (DryRun) [reply DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py]: (y)

**Comment by deguio** [comment]
@DryRun 
this PR should be closed.
thanks.

**Comment by DryRun** [comment]
@deguio Right, thanks!

**Pull request closed without merging**